languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -61277,4 +61277,641 @@ kremlnahem	kremlnah	ADJ:DAT:SIN:NEU:GRU:SOL
+editiert	editieren	VER:PA2:SFT
+editierend	editieren	VER:PA1:SFT
+editiere	editieren	VER:1:SIN:KJ1:SFT
+editierst	editieren	VER:2:SIN:KJ1:SFT
+editiert	editieren	VER:3:SIN:KJ1:SFT
+editieren	editieren	VER:1:PLU:KJ1:SFT
+editiert	editieren	VER:2:PLU:KJ1:SFT
+editieren	editieren	VER:3:PLU:KJ1:SFT
+editierte	editieren	VER:1:SIN:KJ2:SFT
+editiertest	editieren	VER:2:SIN:KJ2:SFT
+editierte	editieren	VER:3:SIN:KJ2:SFT
+editierten	editieren	VER:1:PLU:KJ2:SFT
+editiertet	editieren	VER:2:PLU:KJ2:SFT
+editierten	editieren	VER:3:PLU:KJ2:SFT
+verimpfen	verimpfen	VER:INF:SFT
+verimpf	verimpfen	VER:1:SIN:PRÄ:SFT
+verimpfe	verimpfen	VER:1:SIN:PRÄ:SFT
+verimpfst	verimpfen	VER:2:SIN:PRÄ:SFT
+verimpft	verimpfen	VER:3:SIN:PRÄ:SFT
+verimpfen	verimpfen	VER:1:PLU:PRÄ:SFT
+verimpft	verimpfen	VER:2:PLU:PRÄ:SFT
+verimpfen	verimpfen	VER:3:PLU:PRÄ:SFT
+verimpfen	verimpfen	VER:IMP:PLU:SFT
+verimpfe	verimpfen	VER:IMP:SIN:SFT
+verimpf	verimpfen	VER:IMP:SIN:SFT
+verimpft	verimpfen	VER:IMP:PLU:SFT
+verimpfte	verimpfen	VER:1:SIN:PRT:SFT
+verimpftest	verimpfen	VER:2:SIN:PRT:SFT
+verimpfte	verimpfen	VER:3:SIN:PRT:SFT
+verimpften	verimpfen	VER:1:PLU:PRT:SFT
+verimpftet	verimpfen	VER:2:PLU:PRT:SFT
+verimpften	verimpfen	VER:3:PLU:PRT:SFT
+verimpft	verimpfen	VER:PA2:SFT
+verimpfend	verimpfen	VER:PA1:SFT
+verimpfe	verimpfen	VER:1:SIN:KJ1:SFT
+verimpfst	verimpfen	VER:2:SIN:KJ1:SFT
+verimpft	verimpfen	VER:3:SIN:KJ1:SFT
+verimpfen	verimpfen	VER:1:PLU:KJ1:SFT
+verimpft	verimpfen	VER:2:PLU:KJ1:SFT
+verimpfen	verimpfen	VER:3:PLU:KJ1:SFT
+verimpfte	verimpfen	VER:1:SIN:KJ2:SFT
+verimpftest	verimpfen	VER:2:SIN:KJ2:SFT
+verimpfte	verimpfen	VER:3:SIN:KJ2:SFT
+verimpften	verimpfen	VER:1:PLU:KJ2:SFT
+verimpftet	verimpfen	VER:2:PLU:KJ2:SFT
+verimpften	verimpfen	VER:3:PLU:KJ2:SFT
+verkommen	verkommen	VER:INF:SFT
+verkomm	verkommen	VER:1:SIN:PRÄ:SFT
+verkomme	verkommen	VER:1:SIN:PRÄ:SFT
+verkommst	verkommen	VER:2:SIN:PRÄ:SFT
+verkommt	verkommen	VER:3:SIN:PRÄ:SFT
+verkommen	verkommen	VER:1:PLU:PRÄ:SFT
+verkommt	verkommen	VER:2:PLU:PRÄ:SFT
+verkommen	verkommen	VER:3:PLU:PRÄ:SFT
+verkommen	verkommen	VER:IMP:PLU:SFT
+verkomme	verkommen	VER:IMP:SIN:SFT
+verkomm	verkommen	VER:IMP:SIN:SFT
+verkommt	verkommen	VER:IMP:PLU:SFT
+verkommte	verkommen	VER:1:SIN:PRT:SFT
+verkommtest	verkommen	VER:2:SIN:PRT:SFT
+verkommte	verkommen	VER:3:SIN:PRT:SFT
+verkommten	verkommen	VER:1:PLU:PRT:SFT
+verkommtet	verkommen	VER:2:PLU:PRT:SFT
+verkommten	verkommen	VER:3:PLU:PRT:SFT
+verkommt	verkommen	VER:PA2:SFT
+verkommend	verkommen	VER:PA1:SFT
+verkomme	verkommen	VER:1:SIN:KJ1:SFT
+verkommst	verkommen	VER:2:SIN:KJ1:SFT
+verkommt	verkommen	VER:3:SIN:KJ1:SFT
+verkommen	verkommen	VER:1:PLU:KJ1:SFT
+verkommt	verkommen	VER:2:PLU:KJ1:SFT
+verkommen	verkommen	VER:3:PLU:KJ1:SFT
+verkommte	verkommen	VER:1:SIN:KJ2:SFT
+verkommtest	verkommen	VER:2:SIN:KJ2:SFT
+verkommte	verkommen	VER:3:SIN:KJ2:SFT
+verkommten	verkommen	VER:1:PLU:KJ2:SFT
+verkommtet	verkommen	VER:2:PLU:KJ2:SFT
+verkommten	verkommen	VER:3:PLU:KJ2:SFT
```

Comment:
Die Formen von "verkommen" erscheinen mir nicht korrekt: er ~verkommte~ verkam

Lässt sich bestimmt einfacher als Eintrag ver_kommen in `spelling.txt` einpflegen. Dasselbe gilt für ver_impfen